src/table/src/metadata.rs

```diff
@@ -208,9 +210,34 @@ impl TableMeta {
                     .next_column_id(self.next_column_id);
                 Ok(meta_builder)
             }
+            AlterKind::ChangeTableOptions { options } => self.change_table_attrs(options),
         }
     }
 
+    /// Creates a [TableMetaBuilder] with modified table options.
+    fn change_table_attrs(
+        &self,
+        requests: &[ChangeTableOptionRequest],
+    ) -> Result<TableMetaBuilder> {
```

Comment:
Can we rename to `change_table_options`? We didn't mention attrs.
```suggestion
    fn change_table_options(
        &self,
        requests: &[ChangeTableOptionRequest],
    ) -> Result<TableMetaBuilder> {
```

Reply:
> Can we rename to `change_table_options`? We didn't mention attrs.

fixed in https://github.com/GreptimeTeam/greptimedb/commit/dce487e66e70eb38617be606b3fcbbb74b809111#diff-839e23ed322c90eacc42fdddeb1f7108335e6434eba0a7a1c6dbc42052a8b9e9R396